langwatch/src/components/MainMenu.tsx

```diff
@@ -436,14 +353,16 @@ const SideMenuLink = ({
   return (
     <Tooltip
       content={label}
-      positioning={{ placement: "right" }}
-      disabled={isExpanded}
+      positioning={{
+        placement: "top",
+      }}
+      disabled={size !== "sm"}
+      openDelay={0}
     >
       <Link
-        role="group"
         variant="plain"
         width="full"
-        paddingX={6}
+        paddingX={size === "sm" ? 0 : 4}
```

Comment:
This pattern starts to become messy, especially if we add a new size. Consider a base with a spread override object.